src/components/MovieForm/MovieForm.js

```diff
@@ -9,18 +12,50 @@ import './MovieForm.scss';
 export default function MovieForm(props) {
   const {
     title,
+    onSubmit,
     onClose,
-    movie: initMovie,
+    movie,
   } = props;
+  const dispatch = useDispatch();
 
-  const [movie, setMovie] = useState(initMovie);
-
-  const onChange = (e) => {
-    setMovie({
-      ...movie,
-      [e.target.id]: e.target.value,
-    });
-  };
+  const formik = useFormik({
+    initialValues: {
+      title: movie.title,
+      release_date: movie.release_date,
+      poster_path: movie.poster_path,
+      vote_average: movie.vote_average,
+      genres: movie.genres,
+      runtime: movie.runtime || 0,
+      overview: movie.overview,
+    },
+    validationSchema: Yup.object({
```

Comment:
Can we move this `validationSchema` to its separate module to keep things a bit more readable?

Reply:
Moved to utiils.js. But maybe you can suggest a better place?

Reply:
Another relatable place for that could be in `configs/schemas.js`. For example:

![image](https://github.com/f8B23Lef/reactJS-course-project/assets/60813978/f44b35dd-7462-46b1-84a1-28c183fcb43e)